docs/design.md

```diff
@@ -1195,6 +1195,123 @@ it discovers differences, it reconfigures ranges in the same way
 that it rebalances away from busy nodes, via special-case 1:1
 split to a duplicate range comprising the new configuration.
 
+# SQL
+
+Each node in a cluster can accept SQL client connections.  It supports
+the PostgreSQL wire protocol, to enable reuse of native PostgreSQL
+client drivers. Connections using SSL and authenticated using client
+certificates are supported and even encouraged over unencrypted
+(insecure) and password-based connections.
+
+Each connection is associated with a SQL session which holds the
+server-side state of the connection. Over the lifespan of a session
+the client can send SQL to open/close transactions, issue statements
+or queries or configure session parameters, much like with any other
+SQL database.
+
+## Language support
+
+CockroachDB also attempts to emulate the flavor of SQL supported by
+PostgreSQL, although it also diverges in significant ways:
+
+- CockroachDB exclusively implements MVCC-based consistency for
+  transactions, and thus only supports SQL's isolation levels SNAPSHOT
+  and SERIALIZABLE.  The other traditional SQL isolation levels are
+  not supported and there is no plan to implement them in the future.
```

Comment:
They are supported but internally always map to either SNAPSHOT or SERIALIZABLE.